protect-api/README.md

```diff
@@ -0,0 +1,113 @@
+$ curl http://localhost:8080 -H "Authorization: Bearer {valid-access-token}"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+<!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Java 8+
+- Maven
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                            | Notes                                                                            |
+|:------------------------------:|:---------------------------------------------------------------------|:---------------------------------------------------------------------------------|
+| **Name**                       | `java-api`                                                           | Suggested value for this sample. <br/> You can change the app name at any time.  |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                 |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                               |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Update code sample with app registration values
+
+Open the [_application.yml_](src/main/resources/application.yml) file and modify the three Azure Active Directory configuration properties using the values from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app).
+
+```yaml
+# 'Tenant ID' of your Azure AD instance - this value is a GUID
+tenant-id: value-here
+
+# 'Application ID URI' of app registration in Azure portal - this value typically starts with api://
+app-id-uri: value-here
+```
+
+### 3. Install package(s) and compile application

```

Comment:
I'm less convinced now that a separate **Install packages** section adds value because so many (most? all?) samples use a single command to install all their dependencies.

The original intent of this section was to provide the install command that included the auth package name, but with `dotnet`, `mvn`, `npm`, and `pip` taking care of it behind the scenes with `install.txt` and similar batch facilities, that intent isn't being realized (which is fine because it's devlang/framework-idiomatic way to go).

Should probably collapse it all into **Run the application**, and we'll (content devs) figure out another way to communicate/highlight the package install command for just the auth librar[y|ies] used.

Reply:
Yeah, they have been pretty anemic and tiny.  I'm for collapsing them, reducing the vertical scroll on the page a bit.